packages/taquito/src/estimate/rpc-estimate-provider.ts

```diff
@@ -153,6 +158,7 @@ export class RPCEstimateProvider extends OperationEmitter implements EstimationP
         'storage_size' in result && 'global_address' in result
           ? Number(result.storage_size) || 0
           : 0;
+      totalStorage += 'originated_rollup' in result ? tx_rollup_origination_size : 0;
```

Comment:
could you give a brief breakdown to what this line is doing?

Reply:
Originating a rollup has a storage cost on the blockchain. We need to take into account the rollup origination size (a constant) to calculate the right storageLimit.